code/datums/components/stationstuck.dm

```diff
@@ -0,0 +1,41 @@
+//very similar to stationloving, but more made for mobs and not objects. used on derelict drones currently
+/datum/component/stationstuck
```

Comment:
can this not be a subtype of stationloving?

Reply:
stationloving having a clear purpose for station important objectives is good, having it also cram in mob code that then wouldn't act the same as stationloving is weird and bad